pontoon/base/static/js/table.js

```diff
@@ -23,8 +23,12 @@ var date_formatter = new Intl.DateTimeFormat('en-GB', {
 $('body').on('mouseenter', '.latest-activity .latest time', function() {
   var $element = $(this);
 
+  // Translation Preview (needed for FTL)
+  var translationData = $element.data('translation'),
+      translationPreview = Pontoon.fluent.getSimplePreview({string: translationData}, translationData, {format: 'ftl'});
+
```

Comment:
Retrieving translation format is expensive ATM, so we assume all translations are in the FTL format. That way `getSimplePreview` will return simplified previews of FTL translations and fallback to the default presentation of all other translations.